core/ledger/kvledger/txmgmt/statedb/statecouchdb/couchdbutil.go

```diff
@@ -134,19 +146,50 @@ func createCouchDatabase(couchInstance *couchInstance, dbName string) (*couchDat
 	return &couchDBDatabase, nil
 }
 
+// legacy versions of Couch prior to 3.0.0 did not require a userID and password to be configured when starting
+// the database. If the _users table did not exist in Couch < 3.0 the peer startup process created the _users
+// table. In 3.0.0 the _users table cannot be modified directly. This function can be used to determine if legacy (< 3.0.0)
+// operations can and should be performed.
+func checkLegacyCouchDBVersion(couchVersion string) (bool, error) {
+	couchSemVersion, err := version.NewSemver(couchVersion)
+	if err != nil {
+		return false, errors.Wrap(err, "error parsing CouchDB semver")
+	}
+	gateSemVersion, err := version.NewSemver("3.0.0")
```

Comment:
Shouldn't it be `3.1.0`?
<img width="565" alt="Screenshot 2020-06-17 at 9 04 34 AM" src="https://user-images.githubusercontent.com/8362882/84852114-9bd23f00-b079-11ea-8f3d-ed0863b7aa3a.png">